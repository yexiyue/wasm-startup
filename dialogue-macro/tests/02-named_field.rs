use dialogue_macro::Dialogue;

#[derive(Dialogue)]
enum User {
    Person { name: String, age: u8 },
    Bot { name: String },
}

fn main() {}
