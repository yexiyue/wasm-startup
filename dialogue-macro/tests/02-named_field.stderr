error: only support named struct
 --> tests/02-named_field.rs:3:10
  |
3 | #[derive(Dialogue)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `Dialogue` (in Nightly builds, run with -Z macro-backtrace for more info)
